### PR TITLE
Fix Grid run commands

### PR DESCRIPTION
**Pull request by edenlightning**
Closes #8020

### CODE REVIEW

**Comment by kaushikb11** [comment]
@edenlightning Poonam did similar changes in #8020

**Review by ethanwharris — COMMENTED** [review]
Looks like a duplicate of #8020?

**Review by Borda — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]
Poonam vs Eden let's see who wins the race for the merge ;)

File: docs/source/clouds/cloud_training.rst [hunk 1]
```diff
@@ -1,31 +1,42 @@
...
 
     python my_model.py --learning_rate 1e-6 --layers 2 --gpus 4
 
-And change it to use the grid train command:
+To use the ``grid run`` command:
 
 .. code-block:: bash
 
-    grid train --grid_gpus 4 my_model.py --learning_rate 'uniform(1e-6, 1e-1, 20)' --layers '[2, 4, 8, 16]'
+    grid run --grid_gpus 4 my_model.py --learning_rate 'uniform(1e-6, 1e-1, 20)' --layers '[2, 4, 8, 16]'
```
Comment (ethanwharris) [inline docs/source/clouds/cloud_training.rst:22]: Can now do this:
```suggestion
    grid run --gpus 4 my_model.py --learning_rate 'uniform(1e-6, 1e-1, 20)' --layers '[2, 4, 8, 16]'
```

**Review by ethanwharris — APPROVED** [review]
LGTM :smiley: minor comment

**Pull request merged**